build_tools/azure/install.sh

```diff
@@ -84,7 +84,10 @@ elif [[ "$DISTRIB" == "conda-pip-latest" ]]; then
     setup_ccache
     python -m pip install -U pip
 
-    python -m pip install pandas matplotlib pyamg scikit-image
+    # optionally install scikit-image
```

Comment:
Do we actually want this removed? It will install scikit-image whenever it can, and when the Python 3.9 wheels are released they will be installed, and when Python 3.10 is released and there is no wheels updating this file will not require adding this back.

Reply:
If we always want to install `scikit-image` from binaries (not sure about this), we should specify why.

Maybe something like:

```suggestion
    # Do not build scikit-image from source because it is an optional dependency
```

is clearer.